src/backend/distributed/commands/text_search.c

```diff
@@ -825,16 +825,22 @@ TextSearchDictCommentObjectAddress(Node *node, bool missing_ok)
  * SEARCH CONFIGURATION for which the owner is changed. Optionally errors if the
  * configuration does not exist based on the missing_ok flag passed in by the caller.
  */
-ObjectAddress
+List *
 AlterTextSearchConfigurationOwnerObjectAddress(Node *node, bool missing_ok)
 {
 	AlterOwnerStmt *stmt = castNode(AlterOwnerStmt, node);
 	Relation relation = NULL;
 
 	Assert(stmt->objectType == OBJECT_TSCONFIGURATION);
 
-	return get_object_address(stmt->objectType, stmt->object, &relation, AccessShareLock,
-							  missing_ok);
+	ObjectAddress objectAddress = get_object_address(stmt->objectType, stmt->object,
```

Comment:
for example, here it is needed to have `objectAddressCopy`  because we cannot pass `&objectAddress`